site/app/controllers/MiscController.php

```diff
@@ -397,4 +400,68 @@ private function downloadAssignedZips() {
         readfile("$zip_name");
         unlink($zip_name); //deletes the random zip file
     }
+
+    private function downloadZipStudents() {
+        // security check Disabled as not needed.
```

Comment:
Security check is needed else any other student can download submission of other students via tampering request. Isn't it?